docs/pages/installation/aks.md

```diff
@@ -8,70 +8,85 @@ language: en
+2. If you are using a LoadBalancer, please refer to the [LoadBalancer]({{ site.baseurl }}/installation/platforms/kubernetes#exposing-meshery-serviceloadbalancer) guide for detailed instructions.
+3. Customize your Meshery Provider Callback URL. Meshery Server supports customizing authentication flow callback URL, which can be configured in the following way:
+
+{% capture code_content %}$ MESHERY_SERVER_CALLBACK_URL=https://custom-host mesheryctl system start{% endcapture %}
 {% include code.html code=code_content %}
 
-## Install Meshery on your AKS cluster
+Meshery should now be running in your EKS cluster and Meshery UI should be accessible at the `EXTERNAL IP` of `meshery` service.
+
+## Installation: Using `mesheryctl`
+
+Use Meshery's CLI to streamline your connection to your EKS cluster. Configure Meshery to connect to your EKS cluster by executing:
 
-{% capture code_content %}helm repo add meshery https://meshery.io/charts/
-helm install meshery meshery/meshery --namespace meshery --create-namespace{% endcapture %}
+{% capture code_content %}$ mesheryctl system config aks{% endcapture %}
 {% include code.html code=code_content %}
-<br />
 
-Optionally, Meshery Server supports customizing callback URL for your remote provider, like so:
+Once configured, execute the following command to start Meshery.
 
-{% capture code_content %}helm install meshery meshery/meshery --namespace meshery --set env.MESHERY_SERVER_CALLBACK_URL=https://custom-host --create-namespace{% endcapture %}
+{% capture code_content %}$ mesheryctl system start{% endcapture %}
 {% include code.html code=code_content %}
 
-### Access Meshery UI
+If you encounter any authentication issues, you can use `mesheryctl system login`. For more information, click [here](/guides/mesheryctl/authenticate-with-meshery-via-cli) to learn more.
 
-Access Meshery UI by exposing it as a Kubernetes service or by port forwarding to Meshery UI.
+## Installation: Using Helm
 
-#### [Optional] Port Forward to Meshery UI
+For detailed instructions on installing Meshery using Helm V3, please refer to the [Helm Installation](/installation/helm) guide.
 
-{% capture code_content %}export POD_NAME=$(kubectl get pods --namespace meshery -l "app.kubernetes.io/name=meshery,app.kubernetes.io/instance=meshery" -o jsonpath="{.items[0].metadata.name}")
-kubectl --namespace meshery port-forward $POD_NAME 9081:8080{% endcapture %}
-{% include code.html code=code_content %}
-<br />
+# Post-Installation Steps
+
+## Access Meshery UI
 
-Meshery should now be running in your AKS cluster and the Meshery UI should be accessible at the specified endpoint you've exposed to. Navigate to the `meshery` service endpoint to log into Meshery.
+To access Meshery's UI, please refer to the [instruction](/reference/mesheryctl/system/dashboard) for detailed guidance..
```

Comment:
Good, but you're right that this isn't enough. There are any number of other options available that just this.